### ISSUE TITLE
Use local packages instead of CDN sources in index.html

### ISSUE THREAD

**Issue by ethanchewy** [ethanchewy/PythonBuddy#37]

**Comment by dgmouris** [comment]
I moved these into the static folder although I had a couple of thoughts. 

NPM might be a better option in the future, if you are considering it.

As well I wasn't sure if you wanted to move google fonts locally as well, I just left that as is.

Thanks for your time:)

**Comment by ethanchewy** [comment]
Yeah, I think NPM would be a good option if I use Node.js. Right now, we are mainly just using Flask for the backend and no frontend framework so NPM doesn't make sense at the moment.
